spec/classes/candlepin_spec.rb

```diff
@@ -237,6 +264,18 @@
         it { is_expected.to contain_exec('notification').that_notifies('Service[tomcat]') }
         it { is_expected.to contain_exec('dependency').that_requires('Service[tomcat]') }
       end
+
+      describe 'with manage_ceritifcates false' do
```

Comment:
```suggestion
      describe 'with manage_certificate_files false' do
```

Reply:
Find and replace only works if you spell things correctly the first time!